src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/project.json

```diff
@@ -23,7 +23,7 @@
       }
     }
   },
-  "packInclude": {
+  "packOptions": {
     "Templates/": "Templates/"
```

Comment:
I thought packOptions needs the 
```
"files": {
      "mappings": {
      }
}
```
Also, packOptions will not preserve the directory structure dotnet/cli/issues/2829